terraform/environments/mlra/modules/ecs/main.tf

```diff
@@ -315,11 +315,6 @@ resource "aws_ecs_cluster" "ecs_cluster" {
   )
 }
 
-resource "aws_ecs_cluster_capacity_providers" "ecs_cluster" {
```

Comment:
ECS Cluster Provider is not actually being utilised yet. To be addressed post-migration.